ouroboros-consensus-protocol/ouroboros-consensus-protocol.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version:      3.0
 
 name:                   ouroboros-consensus-protocol
-version:                0.3.0.0
+version:                0.3.1.0
```

Comment:
The `ouroboros-consensus` dependency below is set to `0.3.0.0`

Reply:
Done